odoo/tools/translate.py

```diff
@@ -292,6 +292,49 @@ def parse_xml(text):
 def serialize_xml(node):
     return etree.tostring(node, method='xml', encoding='unicode')
 
+
+MODIFIER_ATTRS = ["invisible", "readonly", "required", "column_invisible", "attrs"]
+def transformer(term_en):
+    def ident(x):
+        return x
+
+    try:
+        orig_node = parse_xml(f"<div>{term_en}</div>")
+    except etree.ParseError:
+        return ident
+
+    if len(orig_node) == 0 or (len(orig_node) == 1 and orig_node[0].tag not in TRANSLATED_ELEMENTS):
+        return ident
+
+    def transform(term):
+        try:
+            new_node = parse_xml(f"<div>{term}</div>")
+        except etree.ParseError:
+            return term
+
```

Comment:
maybe an early return if `len(new_node) != len(orig_node)`
becasue `zip` will stop if one of the `iter` reaches its end.